app/src/main/java/com/example/cachua/musicdemo21/data/repository/MusicRepository.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.cachua.musicdemo21.data.repository
+
+
+import com.example.cachua.musicdemo21.data.model.MusicModel
+import com.example.cachua.musicdemo21.data.model.OnDataLoadedCallback
+import com.example.cachua.musicdemo21.data.source.MusicDatasource
+import com.example.cachua.musicdemo21.data.source.MusicLocalDatasource
+import java.net.MalformedURLException
+
+class MusicRepository : MusicDatasource.Local {
+    private var mMusicLocalDatasoure: MusicLocalDatasource
+    constructor(mMusicLocalDatasoure:MusicLocalDatasource){
+        this.mMusicLocalDatasoure=mMusicLocalDatasoure
```

Comment:
Code xong phải Ctrl + Alt + L để nó auto format lại cho mình chứ
ngoài ra constructor cần tham khảo lại comment của anh Đạt 
```suggestion
        this.mMusicLocalDatasoure = mMusicLocalDatasoure
```